ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -468,6 +470,30 @@ prop_TxSeq_lookupByTicketNo xs =
     txseq = foldl' (TxSeq.:>) TxSeq.Empty
                    (zipWith TxTicket xs (map TicketNo [0..]))
 
+-- | Only finds elements in the sequence
+prop_TxSeq_lookupByTicketNo_sound ::
+    [Positive Int] -> Positive Int -> Property
+prop_TxSeq_lookupByTicketNo_sound raw_xs raw_k =
+    case TxSeq.lookupByTicketNo txseq (TicketNo (toEnum k)) of
+      Just tx' ->
+        label "hit" $
```

Comment:
To increase the hit ratio, try using `Positive (Small Int)` (or is it `Small (Positive Int)`? Only one way round works iirc). That should increase the proportion of collisions.

Reply:
Thanks for the suggestion. I went with `Small` and `abs` to avoid the pitfall.

I also switched to `Set` instead of `scanl` for directness.

It's at about 2/3 misses now.